src/common/utils.ts

```diff
@@ -151,4 +151,7 @@ export const numberToAbbreviation = (numberString: string) => {
   return abbreviation;
 };
 
+// need to check from backend if organization has dynamic role
+export const organizationHasDynamicRole = () => getOrganizationServices('rolesAndPermission');
```

Comment:
@mdshamoon 

Storing this in local storage might have some security implications. Let's chat a bit more if you think otherwise.